### ISSUE TITLE
[REQUEST] Address Input Box

### ISSUE THREAD

**Issue by kgetit** [Styr1x/Browsingway#3]
I am LIKING this and it is so smooth with a GSYNC monitor. As requested, web address input box would be awesome!

**Comment by Styr1x** [comment]
I'll add something akin to that later on, basically a mini browser with all the bells and whistles. Also plan to make links in chat clickable so that it opens inside this mini browser.

Might take a bit of time though, as I'm currently rebuilding all the internals from the old plugin to have a solid foundation to add new features.

**Comment by majorcyto** [comment]
Adding on to this, can we include ublock origin by default (or as a setting) so we can have a nice experience when trying to keep a window up of the wiki or something.

**Comment by Styr1x** [comment]
I don't think that will be possible, extension support in CEF is little to not existent.

**Comment by Styr1x** [comment]
This is now on track for 2.x so closing this.
